<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadDialog</class>
 <widget class="QDialog" name="DownloadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Downloads urls</string>
  </property>
  <layout class="QVBoxLayout" name="lLayout">
   <item>
    <widget class="QLabel" name="lPath">
     <property name="text">
      <string>Path:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lAdd">
     <item>
      <widget class="QLineEdit" name="lePath"/>
     </item>
     <item>
      <widget class="QToolButton" name="tbAdd">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../galleryexplorer.qrc">
         <normaloff>:/res/resources/add.png</normaloff>:/res/resources/add.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbDelete">
       <property name="text">
        <string>Del</string>
       </property>
       <property name="icon">
        <iconset resource="../../galleryexplorer.qrc">
         <normaloff>:/res/resources/del.png</normaloff>:/res/resources/del.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="lwUrls"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lButtons">
     <item>
      <widget class="QPushButton" name="pbDownload">
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="wBtnFree" native="true"/>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../galleryexplorer.qrc"/>
 </resources>
 <connections/>
</ui>
